disgo.go

```diff
@@ -0,0 +1,22 @@
+// Package disgo is a logging and prompting library for
+// modern command line interfaces.
+//
+// It is NOT a substitute for a structured logger
+// in an application or a microservice, since it is
+// completely centered around user interaction and
```

Comment:
It does not provide structured logging and is not built with performance in mind.

Reply:
Ah yes, performance is also an important thing to mention. Thanks for the feedback 🙏🏽